contribs/gmf/src/editing/Snapping.js

```diff
@@ -562,6 +573,17 @@ EditingSnappingService.prototype.loadItemFeatures_ = function(item) {
 };
 
 
+/**
+ * Change the order of xy. This is to deal with the GML output format
+ * from qgisServer that returns inverted coordinates
+ * https://gis.stackexchange.com/questions/296239/gml-axis-order-problem-in-qgis
+ * Loops over all coordinates and only swap X and Y
+ * @private
```

Comment:
Please Add tag `@param` and `@return`
And could you use words instead of `c` ? That's hard to maintain.